src/content/en/tools/lighthouse/audits/description.md

```diff
@@ -0,0 +1,49 @@
+project_path: /web/tools/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Reference documentation for the "Document Does Not Have A Meta Description" Lighthouse audit.
+
+{# wf_updated_on: 2018-02-14 #}
+{# wf_published_on: 2018-02-14 #}
+{# wf_blink_components: Platform>DevTools #}
+
+# Document Does Not Have A Meta Description  {: .page-title }
+
+## Overview {: #overview }
+
+Descriptions can be displayed in Google's search results. High-quality, unique descriptions
+can make your results more relevant to search users and can increase your search traffic.
+
+## Recommendations {: #recommendations }
+
+* Add a description tag to the `<head>` of each of your pages.
+
+    <pre>
```

Comment:
`<pre>` should include `class="prettyprint"`

Reply:
Style guide says it can be omitted in cases where it doesn't make sense. When I staged it locally, the pretty-print appeared to have no effect.

Reply:
I know our dev server forces it in, I don't remember exactly what DevSite does, which is why I suggest including it.

Reply:
Done